### DISCUSSION TITLE
Adding another language

### DISCUSSION THREAD

**Discussion by EasyBabi** [SnosMe/awakened-poe-trade#1249] [Ideas]
Hi SnosMe,

Happy New Year! I've been using your Awakened POE Trade ever since I started POE. Luckily, it helped me enjoy the game even better. Now, I am in South Korea and got to know that Korean is not supported (I was playing in English lang when I was in US). I saw you have no reason to add another language in FAQ, but I am certain that KR users will go crazy about the change, if it ever happens. I am definitely willing to help you adding Korean language, if you need any translation. Please let me know how you think.

**Comment by SnosMe** [comment]
Happy New Year. Thank you for reminding about this :)

**Comment by jhmo96** [comment]
Yes please. I even using 3.19 version (which supports Korean by someone's patch) of it.
It works but can't search for new items. and still it's most best trade support app so I can't stop using it.
It will be very appreciated if you support Korean language.
Thanks for everything SnosMe.

**Reply by SnosMe** [reply]
App supports Korean now, but UI is not translated.

**Reply by EasyBabi** [reply]
do you need any help on translation? I am willing to help.
Please ping @easybabi in game or email easybabi123@gmail.com, if needed.

**Reply by SnosMe** [reply]
Basically need to translate this file https://github.com/SnosMe/awakened-poe-trade/blob/master/renderer/public/data/en/app_i18n.json

Translate as much as you want, feel free to skip entries if you don't know context in which the string is used.

Then drop file here ;)
![image](https://github.com/SnosMe/awakened-poe-trade/assets/4292308/40b904a3-6d0b-4201-bd85-7564dcd1af1d)

**Reply by EasyBabi** [reply]
working on it. I will leave reply when I am done with translation

**Reply by EasyBabi** [reply]
[app_i18n edited by EasyBabi.json](https://github.com/SnosMe/awakened-poe-trade/files/13983296/app_i18n.edited.by.EasyBabi.json)

@SnosMe 

I finished the first sprint. It was not that hard to understand where each string is used. I just wanted to make sure they do not take too much space when it comes to shown in Korean. Also, I did not # for some lines because I feel like it is better to be in English instead of translating in Korean. If you have any questions, please let me know so that we can go over together. 

Things to notice
1) \ back slash does not work on my PC environment ( unfortunately, I do not have visual studio or so installed). Instead, it is shown as KRW (currency) indicator (ln 2 is great example). Please keep it mind when you work on.

2) there are notes I left in English after # bc it was not certain that how it will be shown in actual application. Again, we can go over this when you work on it. 

3) I found that there is an issue when trading. So, when I get a whisper for item, I remember awakened poe trade shows which item buyer wants (like highlighting in stash grid). I am not sure the issue will be solved when translation work is done. 

please let me know. ty mate

**Reply by SnosMe** [reply]
I've integrated your changes in the app, should be now easier to spot what needs attention.
https://github.com/SnosMe/awakened-poe-trade/blob/master/renderer/public/data/ko/app_i18n.json

**Reply by EasyBabi** [reply]
Thanks for the update. It looks way easier to read and edit. I will run through the app so that I can find area that I need to make changes. I will leave comment, if there is any change. Cheers!

**Comment by eddiehuaso** [comment]
what need translate to use awakened poe trade in spanish ?
